src/kvstore/raftex/RaftPart.cpp

```diff
@@ -1395,6 +1395,14 @@ void RaftPart::processAskForVoteRequest(
             });
     }
 
+    if (req.get_last_log_id() < committedLogId_) {
```

Comment:
why not use req.get_committed_log_id()

Reply:
I think leader's lastLogId is greater than its commitLogId, and we don't have a commitLogId in `AskForVoteRequest`, do you think it is necessary to add one?

Reply:
>  leader's lastLogId is greater than its commitLogId

Not exactly,  when catching up data,  maybe lastLogId is less than commitLogId

Reply:
You're right.